django/forms/fields.py

```diff
@@ -204,6 +204,19 @@ def has_changed(self, initial, data):
         data_value = data if data is not None else ''
         return initial_value != data_value
 
+    def bind_to_form(self, form, name):
```

Comment:
I wonder if we might add `**kwargs` to the signature of this method and `BoundField` to allow for future expansion? (In the past, there have been some instances where we've wanted to add additional parameters and we have to add non-trivial backwards compatibility shims to allow that.)